ccmlib/utils/download.py

```diff
@@ -143,10 +144,15 @@ def download_version_from_s3(url: str, target_path: str, verbose=False):
     try:
         metadata = s3_client.head_object(Bucket=bucket_name, Key=download_path)
     except botocore.client.ClientError as ex:
-        if 'Not Found' in str(ex):
+        error_message = ex.response['Error']['Message']
```

Comment:
could you please revisit my previous comment, and review the document. for a sample of how the error is examined, see https://github.com/aws/aws-cli/blob/855cd0ddb3c7fd47cab02a390281f29cc9bc744c/awscli/customizations/s3uploader.py#L123-L127

i think the "Message" field is supposed to be consumed by human being, not to be checked by an application.